### DISCUSSION TITLE
请问编译lean最稳的环境是？

### DISCUSSION THREAD

**Discussion by yayyayyay7** [coolsnowwolf/lede#10810] [Q&A / 快速问答]
虚拟下安装ubuntu？wsl2? github actions? 第一个试过几次都报错。第二个正在试

**Comment by gito10** [comment]
报错不是环境的问题，应该是你自己没配置好。你说的3种都试过，不过自己有电脑的话还是用自己电脑好一点。现在用的是wsl2，原来是hyper-v装的Ubuntu，感觉wsl2好像是要慢一些，不过wsl2的机器配置比装虚拟机的那个要低很多。还没在同一台电脑上试过。不过如果电脑没有其他人要同时使用的话，装个多系统或者虚拟磁盘启动，直接真机启动应该是最快的，毕竟没有win系统的额外开销。

**Comment by fengqi** [comment]
kvm下安装的ubuntu 20.04，很稳，我设置的定时拉代码，定时编译，跑很久了没出错过

**Comment by danuonuo** [comment]
ubuntu吧

**Comment by 8680** [comment]
Ubuntu
